devtools/plugins/desktop/profiler/README.md

```diff
@@ -0,0 +1,39 @@
+# @player-tools/devtools-profiler-web-plugin
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](./LICENSE)
+
+This package provides a basic devtools profiler plugin for the Player UI.
+
+## Overview
```

Comment:
This should get fleshed out with more info

Reply:
very cool stuff though 👍👍

maybe the description can be 

"The profiler can help analyze and optimize player code and render times. It can show functions, and processes as the code runs to analyze any pain points."